pages/search/search.jsx

```diff
@@ -152,6 +152,7 @@ class Search extends React.Component {
         <HelpDropdown 
           value={  this.state.helpType } 
           helpType={ (event) => this.handleHelpChange(event)} 
+          specialClass="help-dropdown"
```

Comment:
We don't need this props. Here's the only place in the codebase that uses `<HelpDropdown />` so `help-dropdown` should just be the default class that comes with the HelpDropdown component.